{
	"threads": 1,
	"sample_only": [
		"mozilla-aurora",
		"try"
	],
	"elasticsearch": {
		"host": "http://localhost",
		"port": 9200,
		"index": "unittest",
		"type": "test_result",
		"timeout": 300,
		"schema": {
			"$ref": "//../../schema/test_results.json"
		},
		"debug": false,
		"limit_replicas": true
	},
	"source": {
		"$ref": "file://~/private.json#aws_credentials",
		"bucket": "active-data-test-result-dev",
		"key_format": "a:b.c"
	},
	"work_queue":{
		"name": "active-data-index-unit-dev",
		"debug": true,
		"timeout": "10second",
		"$ref": "file://~/private.json#aws_credentials"
	},
	"constants": {
		"pyLibrary.env.http.default_headers": {
			"Referer": "https://wiki.mozilla.org/Auto-tools/Projects/ActiveData"
		}
	},
	"debug": {
		"cprofile": {
			"enabled": true,
			"filename": "./results/cprofile.tab"
		},
		"log": [
			{
				"log_type": "console"
			},
			{
				"log_type": "email",
				"from_address": "klahnakoski@mozilla.com",
				"to_address": "klahnakoski@mozilla.com",
				"subject": "[ALERT][DEV]Problem in Push to ES",
				"$ref": "file://~/private.json#email"

			}
		]
	}
}
